infrastructure/modules/key_vault/main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
```

Comment:
Can locals be moved to a `locals.tf` file to keep consistent with the AKS module?